.github/scripts/mr_publish_results.py

```diff
@@ -44,6 +44,7 @@
     "accelerator_type": "ACCELERATOR_TYPE",
     "type": "TYPE",
     "index_repetition": "INDEX_REPETITION",
+    "host_name": "HOST_NAME",
```

Comment:
What is the purpose of using hostname? 

I think a hostname in this case is kinda useless. The host is ephemeral and the name is random.

Reply:
It can tell if 2 jobs run on the same machine one after the other, e.g. when investigating if something is pre-downloaded when the 2nd job starts. 

I started to do some speed/benchmarking investigations myself now, and I was in a situation where I wanted to group by (DD calls it `avg by`) host name.

It also provides all tags (also the custom metrics) with a sign where this was run, (e.g. if on a K8s runner setup, or on an Azure VM).

> The host is ephemeral and the name is random.

Yeah, it can not really be used to go back to this machine and look at it post-hoc, I agree.